### PR TITLE
SRVCOM-1451: Add serving, security, support, additional admin docs for OSD

### PR DESCRIPTION
**Pull request by abrennan89**
Jira: https://issues.redhat.com/browse/SRVCOM-1451

Applies for OCP 4.9, 4.10+

Direct preview links:
- https://deploy-preview-42307--osdocs.netlify.app/openshift-dedicated/latest/serverless/develop/serverless-applications.html
- https://deploy-preview-42307--osdocs.netlify.app/openshift-dedicated/latest/serverless/develop/serverless-autoscaling-developer.html
- https://deploy-preview-42307--osdocs.netlify.app/openshift-dedicated/latest/serverless/develop/serverless-traffic-management.html
- https://deploy-preview-42307--osdocs.netlify.app/openshift-dedicated/latest/serverless/develop/serverless-configuring-routes.html
- https://deploy-preview-42307--osdocs.netlify.app/openshift-dedicated/latest/serverless/admin_guide/serverless-cluster-admin-eventing.html
- https://deploy-preview-42307--osdocs.netlify.app/openshift-dedicated/latest/serverless/admin_guide/serverless-cluster-admin-serving.html
- https://deploy-preview-42307--osdocs.netlify.app/openshift-dedicated/latest/serverless/admin_guide/knative-serving-cr-config
- https://deploy-preview-42307--osdocs.netlify.app/openshift-dedicated/latest/serverless/admin_guide/serverless-admin-autoscaling.html
- https://deploy-preview-42307--osdocs.netlify.app/openshift-dedicated/latest/serverless/serverless-support.html
- https://deploy-preview-42307--osdocs.netlify.app/openshift-dedicated/latest/serverless/security/serverless-ossm-with-kourier-jwt.html
- https://deploy-preview-42307--osdocs.netlify.app/openshift-dedicated/latest/serverless/security/serverless-custom-domains.html
- https://deploy-preview-42307--osdocs.netlify.app/openshift-dedicated/latest/serverless/security/serverless-custom-tls-cert-domain-mapping.html
- https://deploy-preview-42307--osdocs.netlify.app/openshift-dedicated/latest/serverless/security/serverless-kafka-security.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
LGTM!

File: modules/serverless-creating-channel-admin-web-console.adoc [hunk 1]
```diff
@@ -6,12 +6,25 @@
 [id="serverless-creating-channel-admin-web-console_{context}"]
 = Creating a channel by using the Administrator perspective
 
-If you have cluster administrator permissions, you can create a channel by using the Administrator perspective in the web console.
+ifdef::openshift-enterprise[]
+If you have cluster administrator permissions, you can create a channel by using the *Administrator* perspective in the web console.
+endif::[]
+
+ifdef::openshift-enterprise[]
```
Comment (jrangelramos) [inline modules/serverless-creating-channel-admin-web-console.adoc:13]: same here... probably (?)..
```suggestion
ifdef::openshift-dedicated[]
```

File: modules/serverless-creating-broker-admin-web-console.adoc [hunk 2]
```diff
@@ -6,12 +6,25 @@
 [id="serverless-creating-broker-admin-web-console_{context}"]
 = Creating a broker by using the Administrator perspective
 
-If you have cluster administrator permissions, you can create a broker by using the Administrator perspective in the web console.
+ifdef::openshift-enterprise[]
+If you have cluster administrator permissions, you can create a broker by using the *Administrator* perspective in the web console.
+endif::[]
+
+ifdef::openshift-enterprise[]
```
Comment (jrangelramos) [inline modules/serverless-creating-broker-admin-web-console.adoc:13]: I think here it probably should be...
```suggestion
ifdef::openshift-dedicated[]
```

**Review by jrangelramos — COMMENTED** [review]
Just 2 minor nits. Besides that it looks good to me.

File: modules/serverless-creating-channel-admin-web-console.adoc (hunk 1 above)
Comment (abrennan89) [reply modules/serverless-creating-channel-admin-web-console.adoc:13]: thanks for catching these!

**Pull request merged**